src/components/vulnerability-list/index.tsx

```diff
@@ -0,0 +1,149 @@
+import { Card, Divider, List, Typography } from '@equinor/eds-core-react';
+import * as PropTypes from 'prop-types';
+import ReactMarkdown from 'react-markdown';
+
+import { externalUrls } from '../../externalUrls';
+import {
+  VulnerabilityModel,
+  VulnerabilityModelValidationMap,
+} from '../../models/vulnerability';
+import { formatDateTimeYear } from '../../utils/datetime';
+
+import './style.css';
+
+export interface VulnerabilityListProps {
+  vulnerabilityList: Array<VulnerabilityModel>;
+}
+
+const SeverityIcon = ({ severity }: { severity: string }): JSX.Element => (
+  <span className={`severity-icon severity-icon--${severity.toLowerCase()}`}>
+    <Typography variant="h3" color="white">
+      {severity.substring(0, 1).toUpperCase()}
+    </Typography>
+  </span>
+);
+
+const VulnerabilityMetadata = ({
+  vulnerability,
+}: {
+  vulnerability: VulnerabilityModel;
+}): JSX.Element => (
+  <List className="severity-metadata">
+    <List.Item>
+      <Typography variant="overline">Version</Typography>
+      {vulnerability.version}
+    </List.Item>
+    {vulnerability.cwe?.length > 0 && (
+      <List.Item>
+        <Typography variant="overline">CWE</Typography>
+        <Typography
+          link
+          rel="noopener noreferrer"
+          target="_blank"
+          href={externalUrls.cweVulnerabilityInformation(vulnerability.cwe[0])}
+        >
+          {vulnerability.cwe[0]}
+        </Typography>
+      </List.Item>
+    )}
+    {vulnerability.cve?.length > 0 && (
+      <List.Item>
+        <Typography variant="overline">CVE</Typography>
+        <Typography
+          link
+          rel="noopener noreferrer"
+          target="_blank"
+          href={externalUrls.cveVulnerabilityInformation(vulnerability.cve[0])}
+        >
+          {vulnerability.cve[0]}
+        </Typography>
+      </List.Item>
+    )}
+    {vulnerability.cvss && (
+      <List.Item>
+        <Typography variant="overline">CVSS</Typography>
+        {vulnerability.cvss}
+      </List.Item>
+    )}
+    {vulnerability.publishedDate && (
+      <List.Item>
+        <Typography variant="overline">Published Date</Typography>
+        {formatDateTimeYear(vulnerability.publishedDate)}
+      </List.Item>
+    )}
+  </List>
+);
+
+const VulnerabilityItem = ({
+  vulnerability,
+}: {
+  vulnerability: VulnerabilityModel;
+}): JSX.Element => (
+  <Card>
```

Comment:
Add class to use for styling
```suggestion
  <Card className="vulnerability-list">
```

Reply:
fixed by refactoring component